src/uielement_button.js

```diff
@@ -0,0 +1,73 @@
+/**
+A UIElement that shows a button.
+The provided callback function gets called if a onclick event occurs on this button.
+
+@class UIElementButton
+@augments UIElement
+
+@param {string} [className] CSS class
+@param {string} caption Caption of the button
+@param {method} actionFunction Callback function for onclick event
+*/
+function UIElementButton(className, caption, actionFunction) {
+    UIElement.call(this);
+
+    this.className = className;
+    this.caption = caption;
+    this.actionFunction = actionFunction;
+
+    this.node = null;
+    this.button = null;
```

Comment:
button is never used outside of createUI() and thus not required to be an attribute.

Reply:
`setEnabled()` uses the button. Whether it is necessary to implement `setEnabled()`, I don't know.